### ISSUE TITLE
(Fabric 1.20.1) Paper walls and doors are uncraftable

### ISSUE THREAD

**Issue by YouYoumuKonpaku** [Poulpynou/dawnoftimebuilder#56]
MC Ver. 1.20.1
Fabric Ver. 0.16.5
Dawn of Time Ver. Fabric-1.5.8

Paper walls and other items that require sticks to craft are uncraftable for some reason. JEI displays the recipes without the sticks. I should also add that every item that needs sticks is not craftable, not just paper walls and doors.
Not really sure what's causing this, this doesn't seem to be a mod conflict. Here are the mods I had installed.
JEI Ver. 12.19.5.99
Fabric API Ver. 0.92.2
Fusion Ver. 1.1.1
Dawn of Time Ver. Fabric-1.5.8

This is what JEI displays
![2024-09-27_17 22 24](https://github.com/user-attachments/assets/1ac7b1d0-f47c-481e-99db-e789184d6a51)

This either isn't the right recipe, or for some reason the recipe is glitched.
![2024-09-27_17 22 27](https://github.com/user-attachments/assets/1ef7b224-a909-42a0-ae20-c4d7925ee16e)
I placed sticks in the recipes I thought needed them, but nothing happened.

**Comment by Behavedgurl** [comment]
Same here.. as well with the paper floor lamp..

**Comment by YouYoumuKonpaku** [comment]
> Same here.. as well with the paper floor lamp..

Yeah, it seems to be every recipe that requires sticks or some other wood item. I assume it's some fabric bug that needs to be ironed out.

**Comment by TheGoldenWorld** [comment]
Hello there, thanks for your bug report guys, this new 1.5.8 received a lot of internal changes to support multi loader versions (that's why the mod is now available on Fabric) . If you see more, do not hesitate to let us know 🙏

**Comment by YouYoumuKonpaku** [comment]
> Hello there, thanks for your bug report guys, this new 1.5.8 received a lot of internal changes to support multi loader versions (that's why the mod is now available on Fabric) . If you see more, do not hesitate to let us know 🙏

Will do. Everything else seems to be working as intended, but I'll make sure to make a new issue if I find anything.

**Comment by HydrolienF** [comment]
It can probably be fix by replacing `"tag": "forge:rods/wooden"` by `"item": "minecraft:stick"` in each recipes. Fabric probably doesn't have any forge related tag.
There is also some other forge tag that would need to be replace as `"forge:dyes/black"` or `"tag": "forge:ingots/iron"`.

**Comment by TheGoldenWorld** [comment]
It has been fixed in the last 1.5.9 version of DoT for Fabric. Thanks for your bug reports boyz and thx Hydro for your pull request ! I close this issue
